### ISSUE TITLE
User can unmark a task.

### ISSUE THREAD

**Issue by JohnLeeJH** [JohnLeeJH/Daily-task-manager-app#5]
Motivation
User can unmark a task from completed mark. If they want to re-open the task.

Implements
- [x] Enhance eventListener to reverse the complete check mark

Note: Above feature was done along with previous feature(User can mark completed task)
I'm doing minor refactoring for some previous feature along with this feature.
- [x] User can add new task only after typing in new text. (Means, empty text line can not be added)
Finished feature
![image](https://cloud.githubusercontent.com/assets/26069024/25963139/1d5a912a-3635-11e7-98f3-427732ea034c.png)
